drivers/wifi/nrfwifi/Kconfig.nrfwifi

```diff
@@ -555,6 +555,23 @@ config NET_TC_TX_COUNT
 
 endif # NETWORKING
 
+
+# These numbers were derived from tests using specific configurations for TCP and UDP
+# with variable data sizes. We adjusted pool sizes to match throughput levels from
+# previous fixed data tests.
+
+choice NET_PKT_DATA_ALLOC_TYPE
+	default NET_BUF_VARIABLE_DATA_SIZE
+endchoice
+
+if NET_BUF_VARIABLE_DATA_SIZE
+config NET_PKT_BUF_RX_DATA_POOL_SIZE
+	default 32000
+
+config NET_PKT_BUF_TX_DATA_POOL_SIZE
+	default 55000
+endif
```

Comment:
should we protect this with a Kconfig? with default still being variable buffers?